tests/sentry/interfaces/test_stacktrace.py

```diff
@@ -410,18 +410,6 @@ def test_emoji_culprit(self):
         ]))
         assert stacktrace.get_culprit_string(platform='javascript') == u'\U0001f60d(\U0001f62d)'
 
-    def test_exclude_libswiftCore_from_in_app(self):
```

Comment:
This test was bullshit. It tested something completely wrong so it's removed. It did not test what the test name suggests.